### ISSUE TITLE
queries about Lexonomy features

### ISSUE THREAD

**Issue by iztokkosem** [elexis-eu/lexonomy#278]
Here we are recording some queries about Lexonomy features that are currently unclear, while preparing the editor. The answers should form part of documentation, so please do not close this ticket.

Question 1:
- What is the difference between two options in crosslinking settings? (one is for "clickable", the other for "External URL)

Question 2:
- What does it mean to display "caption" or "value" (it seems to be linked with dropdown menus, e.g. at part-of-speech category)?

**Comment by rambousek** [comment]
1) clickable crossreference - element value is word(s). In preview, this element is displayed as a link. When user clicks the link, Lexonomy will search for the element value and display results.

External URL - element value is URL. In preview, URL is displayed as link.

2) in structure, when specifying content "value from list", you enter value (stored in XML) and user-friendly label/caption for each item. In formatting, you man select which of them to display

**Comment by iztokkosem** [comment]
> 1. clickable crossreference - element value is word(s). In preview, this element is displayed as a link. When user clicks the link, Lexonomy will search for the element value and display results.
> 
OK, to be clear: if I enter "chair" in the element, "clickable cross-reference" makes it into a search "chair" (exact match?) in that dictionary. So it is not cross-reference per say, but more a link to a customised search.
The second one is a proper cross-reference, which can lead to an entry in that dictionary, or anywhere externally (Lexonomy or out of Lexonomy).

> External URL - element value is URL. In preview, URL is displayed as link.
> 
> 2. in structure, when specifying content "value from list", you enter value (stored in XML) and user-friendly label/caption for each item. In formatting, you man select which of them to display

So to exemplify - I can use "adj." for the value, and caption "adjective". If chosing the Caption option, adjective is displayed, if not, adj. is displayed. Right?

**Comment by rambousek** [comment]
> OK, to be clear: if I enter "chair" in the element, "clickable cross-reference" makes it into a search "chair" (exact match?) in that dictionary. So it is not cross-reference per say, but more a link to a customised search. The second one is a proper cross-reference, which can lead to an entry in that dictionary, or anywhere externally (Lexonomy or out of Lexonomy).

Yes, it starts search for "chair" (headwords starting with chair)

> So to exemplify - I can use "adj." for the value, and caption "adjective". If chosing the Caption option, adjective is displayed, if not, adj. is displayed. Right?
Exactly
